guide/07-working-with-bigdata/01-introduction.ipynb

```diff
@@ -180,6 +180,13 @@
    "source": [
```

Comment:
<p>The module verifies that your Enterprise has been properly configured with the&nbsp;<code><a href="https://esri.github.io/arcgis-python-api/apidoc/html/arcgis.geoanalytics.toc.html#is-supported" target="_blank">is_supported()</a></code>&nbsp;function. Once it is confirmed that the Enterprise is configured, the geoanalytics tools work with&nbsp;<a href="http://enterprise.arcgis.com/en/server/10.5/get-started/windows/what-is-a-big-data-file-share.htm" target="_blank">big data file share</a>&nbsp;items registered in the Web GIS and with&nbsp;<a href="https://developers.arcgis.com/rest/services-reference/feature-layer.htm" target="_blank">feature layers</a>.</p><p> </p><ul><li>the&nbsp;<code><a href="https://esri.github.io/arcgis-python-api/apidoc/html/arcgis.geoanalytics.analyze_patterns.html" target="_blank">arcgis.geoanalytics.analyze_patterns</a></code>&nbsp;module provides tools that help you identify, quantify, and visualize spatial patterns in your data.</li><li>the&nbsp;<code><a href="https://esri.github.io/arcgis-python-api/apidoc/html/arcgis.geoanalytics.find_locations.html" target="_blank">arcgis.geoanalytics.find_locations</a></code>&nbsp;module provides tools that help you identify areas or features that meet any number of criteria you specify.</li><li>the&nbsp;<code><a href="https://esri.github.io/arcgis-python-api/apidoc/html/arcgis.geoanalytics.manage_data.html" target="_blank">arcgis.geoanalytics.manage_data</a></code>&nbsp;module provides tools for the day-to-day management of geographic and tabular data.</li><li>the&nbsp;<code><a href="https://esri.github.io/arcgis-python-api/apidoc/html/arcgis.geoanalytics.summarize_data.html" target="_blank">arcgis.geoanalytics.summarize_data</a></code>&nbsp;module provides tools that output descriptive statistics of features and their attributes based on spatial relationships to other features.</li><li>the&nbsp;<code><a href="https://esri.github.io/arcgis-python-api/apidoc/html/arcgis.geoanalytics.use_proximity.html" target="_blank">arcgis.geoanalytics.use_proximity</a></code>&nbsp;module provides tools that define geometries to help analyze which features are near other features.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>There are changes in the paragraph and bullets that still need to be made.</p>